libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/NavigationCamera.java

```diff
@@ -212,6 +221,53 @@ public void showRouteOverview(int[] padding) {
     animateCameraForRouteOverview(routeInformation, padding);
   }
 
+  /**
+   * Animate the camera to a new location defined within {@link CameraUpdate} passed to the
+   * {@link NavigationCameraUpdate} using a transition animation that evokes powered flight.
```

Comment:
Would it make sense to add something along the lines of
> ... that evokes powered flight. If the camera is in a tracking mode, this animation is going to be ignored, or break the tracking, based on the `CameraUpdateMode`.

to each javadoc entry?